alignak/comment.py

```diff
@@ -109,9 +121,19 @@ def __init__(self, params):
         :type expire_time: int
         :return: None
         """
-        super(Comment, self).__init__(params)
-        self.entry_time = int(time.time())
-        self.fill_default()
+        if 'uuid' not in params:
```

Comment:
It's yet managed in AlignakObject, the uuid management with params, we can simplify a little ^^